plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/CodeModernizerBanner.kt

```diff
@@ -42,11 +40,13 @@ class CodeModernizerBanner(val project: Project) : JPanel(BorderLayout()) {
         border = BorderFactory.createEmptyBorder(0, 5, 0, 0)
     }
 
+    private val infoLabelJobId = JBLabel().apply {
```

Comment:
Sorry just for my own understanding, this is for the ux of displaying the job id in transformation hub right